tests/x11/rmt/rmt_chinese.pm

```diff
@@ -34,7 +34,8 @@ sub set_language_to_Chinese {
     x11_start_program('xterm');
     wait_still_screen 2, 2;
     become_root;
-    script_run('yast2 language');
+    script_run('yast2 language', die_on_timeout => 0);
+    assert_screen 'yast2-language', 60;
```

Comment:
You can add a assert_screen to make it more stable, while the issue is for the script_run timeout. So please update the PR description for the root cause and how you fixed it.

Reply:
done, please help to review, thanks.